contrib/znc/files/sysusers.conf

```diff
@@ -0,0 +1,3 @@
+# create _znc user
+
+u _znc - "User for ZNC bouncer" /var/empty /bin/sh
```

Comment:
/var/empty should be /var/lib/znc

and /bin/sh should be /usr/bin/nologin unless the service itself uses the user via su scripts or similar; for actual administration you don't need a shall ehere

Reply:
yup i didnt know `su -s`  was a thing